test/transaction.test.js

```diff
@@ -247,4 +250,32 @@ describe('transactions', function() {
         });
       });
   });
+
+  it('can return promise for commit', function() {
+    const connectorObject = {};
+    connectorObject.commit = function(connection, cb) {
+      return cb(null, 'committed');
+    };
+    const transactionInstance = new Transaction(connectorObject, {});
+    return expect(transactionInstance.commit()).to.eventually.equal('committed');
+  });
+
+  it('can return promise for rollback', function() {
+    const connectorObject = {};
+    connectorObject.rollback = function(connection, cb) {
+      return cb(null, 'rolledback');
+    };
+    const transactionInstance = new Transaction(connectorObject, {});
+    return expect(transactionInstance.rollback()).to.eventually.equal('rolledback');
+  });
+
+  it('can return promise for begin', function() {
+    const connectorObject = {};
+    connectorObject.beginTransaction = function(connection, cb) {
+      return cb(null, 'begun');
+    };
+
+    return expect(
+      Transaction.begin(connectorObject, '')).to.eventually.be.instanceOf(Transaction);
```

Comment:
I think we are usually using slightly different formatting.

LB3 style:

```js
return expect(Transaction.begin(connectorObject, ''))
  .to.eventually.be.instanceOf(Transaction);
```

LB4 style (?)

```js
return expect(
  Transaction.begin(connectorObject, '')
).to.eventually.be.instanceOf(Transaction);
```

Reply:
Okay I'll go with LB3 style 👍